### DISCUSSION TITLE
A memoized async atom in a component will trigger infinite recreation

### DISCUSSION THREAD

**Discussion by uonr** [pmndrs/jotai#1793] [Q&A] [answered]
https://codesandbox.io/s/jotai-sleep-gw6qkp?file=%2Fsrc%2FApp.tsx

Should I not create async atom in the component?

**Comment by dai-shi** [comment, marked as answer]
useMemo will refresh on suspend, so it can't return a stable reference.

**Reply by uonr** [reply]
I also tried useRef and had the same result, I can't seem to find a way to maintain a stable reference

**Reply by dai-shi** [reply]
Yeah, that's how Suspense works.
